orchestra/static/orchestra/todos/todo-qa-list.html

```diff
@@ -0,0 +1,45 @@
+<script type="text/ng-template" id="todoQaItemTemplate">
```

Comment:
high-level: should the qa.directive.* be merged with this so there are less components? unlike React, there's not too much benefit to the multiple nested component approach